tests/k8s/test_k8s.py

```diff
@@ -236,6 +241,26 @@ def test_flow_with_configmap(
         assert doc.tags.get('env') == {'k1': 'v1', 'k2': 'v2'}
 
 
+@pytest.mark.timeout(3600)
+def test_flow_with_namespace(
+    k8s_cluster,
+    k8s_flow_with_namespace,
+    load_images_in_kind,
+    set_test_pip_version,
+    logger,
+):
+    resp = run_test(
+        k8s_flow_with_namespace,
+        endpoint='/namespace',
+        port_expose=9090,
+    )
+
+    docs = resp[0].docs
+    assert len(docs) == 10
+    for doc in docs:
+        assert doc.tags.get('namespace') == 'k8s-flow-with-namespace-ns'
```

Comment:
how do u expect to get this?, u forgot to add an executor folder?

Reply:
this is not what I intended to add, some legacy code. uncommitted